tyk-headless/README.md

```diff
@@ -1,7 +1,21 @@
-## Tyk Community Edition
-Provides a Tyk Installation for Community Edition Gateways
+## Tyk OSS Gateway
```

Comment:
Since the naming has changed. Maybe it's better to change the directory to `tyk-ce` or `tyk-oss`

this may require changes to GitHub Actions.

Reply:
we will but only in v1.0 since it's a breaking change for current users.